### ISSUE TITLE
Rewriting of command help

### ISSUE THREAD

**Issue by ghost** [SimpleServer/SimpleServer#95]
Currently some commands (e.g. `/home help`) have internal help while others don't. All commands will have this internal help which displays all the uses for the command. The parsing of the `help` argument will also be rewritten, so that the command is not executed when the internal help is called.

**Comment by heapoutofspace** [comment]
I don't think this is a good idea. Right now each command defines it's help string which is then used by a global help command. The only thing that could be better is a method in AbstractCommand which only returns said string and can be overriden in commands with the need for a more complex helptext.

So, no `/command help` but better `/help command`

**Comment by ghost** [comment]
Well, OK.
